src/web_interface/components/analysis_routes.py

```diff
@@ -232,6 +233,25 @@ def show_elf_dependency_graph(self, uid: str, root_uid: str):
             colors=colors,
         )
 
+    @staticmethod
+    def _is_link(file_obj: FileObject) -> bool:
+        type_analysis = file_obj.processed_analysis.get('file_type', {}).get('result', {})
+        mime = type_analysis.get('mime')
+        full_type = type_analysis.get('full', '')
+        return mime == 'inode/symlink' and full_type.startswith('symbolic link to')
+
+    def _get_link_target(self, file_obj: FileObject, root_uid: str) -> str | None:
+        if not root_uid:
+            return None
+        try:
+            full_type = file_obj.processed_analysis['file_type']['result']['full']
+            # if FO is a symlink, file_type analysis "full" will be something like "symbolic link to 'busybox'"
+            target_path = full_type.split("'")[1]
+        except (IndexError, KeyError):
+            return None
```

Comment:
Again, we can't really do much about this, but this prone to be broken.
For example, it does not work with the link created by `ln -s "./crash'es" "foo'bar"`.
Furthermore, I'd like to note that this is actually not standard `file` output,
but defined in `src/install/internal_symlink_magic`.